r11701/default_common_inlists/binary/inlist_project

```diff
@@ -51,6 +51,9 @@
    accretor_overflow_terminate = 100d0
    terminate_if_L2_overflow = .false. ! L2 oveflow check changes between Marchant et al. 2016 and Misra et al. 2020 for MS or postMS star respectively, in run_binary_extras.f
 
+   use_other_rlo_mdot = .true. ! to be used for reverse MT with postMS evolution of one star, with Kolb scheme.
```

Comment:
Is it better to add these to the run_binary_extras? After checking if point_mass is indeed 0.0?